### PR TITLE
Add Dockerfile.rhel8

### PR DESCRIPTION
**Pull request by yselkowitz**
A separate file is required, as the changes necessary for building with
RHEL 8 content are not compatible with RHEL 7.

### CODE REVIEW

File: Dockerfile.rhel8 [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+COPY manifests /manifests
+ENV APP_ROOT=/var/lib/tuned
+ENV PATH=${APP_ROOT}/bin:${PATH}
+ENV HOME=${APP_ROOT}
+WORKDIR ${APP_ROOT}
+COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/_output/openshift-tuned /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/cluster-node-tuning-operator/assets ${APP_ROOT}
+COPY --from=tuned   /root/rpmbuild/RPMS/noarch /root/rpms
+RUN INSTALL_PKGS=" \
+     hdparm kernel-tools tuna socat \
```
Comment (jmencak) [inline Dockerfile.rhel8:30]: Let's drop `hdparm`, `kernel-tools` and `tuna` and keep only `socat`.  `socat` is the only real dependency for NTO's operand (openshift-tuned).  `tuna` will be pulled in by tuned profiles that need it by `yum --setopt=tsflags=nodocs -y install /root/rpms/*.rpm` and so will `hdparm` and `kernel-tools` on platforms that ship those packages.  On RHEL 7, there's no package `hdparm` and `kernel-tools` for  s390x.  This seems this is no longer the case for RHEL 8, but let's keep Dockerfiles as much close to each other as possible.  Would you be able to adjust Dockerfile and Dockerfile.rhel7 in a similar way?  Thank you!

Comment (yselkowitz) [reply Dockerfile.rhel8]: In that case, we don't even need a Dockerfile.rhel8.  Modified PR accordingly.

**Comment by jmencak** [comment]
/lgtm
/approve
Thank you!

**Pull request merged**